Expecto/Expecto.fs

```diff
@@ -330,7 +330,7 @@ module Impl =
       >> setField "failed" failed
       >> setField "errored" errored)
 
-
+  let semaphore = obj ()
```

Comment:
Didn't work for me. I think it's because new semaphore is created for the console target https://github.com/logary/logary/blob/master/src/Logary.Facade/Facade.fs#L691, but might be wrong